tests/functional/wdio.conf.js

```diff
@@ -17,6 +17,12 @@ exports.config = {
   // directory is where your package.json resides, so `wdio` will be called from there.
   //
   specs: ["./tests/functional/spec/**/*.js"],
+  suites: {
+    timeout_modal: ["./tests/functional/spec/timeout/*.spec.js"],
+    components: ["./tests/functional/spec/components/**/*.js"],
+    features: ["./tests/functional/spec/features/**/*.js"],
```

Comment:
Oddly, I didn't think this would pick up file within `features` but only folders, but it does seem to work fine with files.

Reply:
Yeah think this picks. up both files and folders 👍